src/main/java/org/sonarqube/A.java

```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
```

Comment:
###AWASQB4Plc2VNCmm4xe4###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_15@6
Complete the task associated to this TODO comment.

Reply:
===CLOSED===
This issue has been resolved by a change in code

Reply:
===CLOSED===
This issue has been resolved by a change in code